tasks/utilities/data_handling/task_fetch_srr_metadata.wdl

```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+task fetch_srr_metadata {
+  input {
+    String sample_accession 
+    String docker = "us-docker.pkg.dev/general-theiagen/biocontainers/fastq-dl:2.0.4--pyhdfd78af_0"
+    Int disk_size = 10
+    Int cpu = 2
+    Int memory = 8
+  }
+  meta {
+    volatile: true
+  }
+
+  command <<< 
+    mkdir -p metadata_output
+    date -u | tee DATE
+
+    # Debug output to show the sample being processed
+    echo "Fetching metadata for sample accession: ${sample_accession}"
+
+    # Use fastq-dl to fetch metadata only
+    fastq-dl --accession ~{sample_accession} --outdir metadata_output --only-download-metadata --verbose
+
+
+    if [[ -f metadata_output/fastq-run-info.tsv ]]; then
+      echo "Metadata written for ${sample_accession}:"
+      echo "TSV content:"
+      cat metadata_output/fastq-run-info.tsv
```

Comment:
Do we want to output this file as well? Maybe not to the table but as a task output?

Reply:
I dont think we would want this as we output this into the log file for debugging anyway